DESCRIPTION

```diff
@@ -24,7 +24,9 @@ Imports:
 	FeatureExtraction,
 	ff,
```

Comment:
Would you do the honor of removing this dependency ;-)

Reply:
Done https://github.com/OHDSI/CohortDiagnostics/pull/49/commits/568851d9a9ac11fd11befbe65097f4200b875af9